cmd/goal/node.go

```diff
@@ -374,3 +394,97 @@ var waitCmd = &cobra.Command{
 		}
 	},
 }
+
+func isValidIP(userInput string) bool {
+	host, port, err := net.SplitHostPort(userInput)
+	if err != nil {
+		return false
+	}
+	if port == "" {
+		return false
+	}
+	if host == "" {
+		return false
+	}
+	return isValidIP(host)
+}
+
+var createCmd = &cobra.Command{
+	Use:   "create",
+	Short: "create a node at the desired data directory for the desired network",
+	Long:  "create a node at the desired data directory for the desired network",
+	Args:  validateNoPosArgsFn,
+	Run: func(cmd *cobra.Command, _ []string) {
+
+		// validate network input
+		validNetworks := map[string]bool{"mainnet": true, "testnet": true, "devnet": true}
```

Comment:
Fine for now - but realized this should really allow anything and just fail if we don't find the genesis.json file for that network.